packages/teams-js/src/private/logs.ts

```diff
@@ -1,6 +1,7 @@
 import { sendMessageToParent } from '../internal/communication';
```

Comment:
Should we also change the name of the namespace and also file? If not, what are the guidelines for what should stay 'logs' and what should change to 'log'?

Reply:
We should not change the name of the namespace. There was a problem with the runtime being inconsistent with the host SDK and the decision was made to update the teams-js SDK to match what is in the host SDK. Although now that i think about it, I think this does set a confusing precedent.

Reply:
We should follow up on clarifying what should stay 'logs' vs 'log'.